catalogs/catalog-fileset-hadoop/build.gradle.kts

```diff
@@ -31,12 +34,12 @@ tasks {
   val copyCatalogLibs by registering(Copy::class) {
     dependsOn(copyDepends, "build")
     from("build/libs_all", "build/libs")
-    into("$rootDir/distribution/package/catalogs/fileset/libs")
+    into("$rootDir/distribution/package/catalogs/hadoop/libs")
```

Comment:
Should we change this `hadoop` to `fileset-hadoop`? All is ok for me. If we add another file system, for example: ceph. Should we use `catalogs/ceph`?

Reply:
I don't want to add "fileset" to the packaged catalog name, we don't have to emphasize the type in package name. Also, if we want to follow the {type}-{provider} pattern, then we need to change all the relational catalogs' package name.

Reply:
OK for me. Just feel Hadoop is a broad concept.  A little concerned about it.

Reply:
Let me think about it.